x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authz/AuthorizationServiceTests.java

```diff
@@ -479,6 +484,46 @@ public void testRemoteIndicesOnlyWorkWithApplicableRequestTypes() throws IOExcep
         verifyNoMoreInteractions(auditTrail);
     }
 
+    public void testUserWithNoRolesOpenPointInTimeWithRemoteIndices() {
+        final boolean hasLocalIndices = randomBoolean();
+        final String[] indices = new String[]{
+            hasLocalIndices ? randomAlphaOfLength(5) : "other_cluster:" + randomAlphaOfLength(5),
+            "other_cluster:" + randomAlphaOfLength(5)
+        };
+        final OpenPointInTimeRequest openPointInTimeRequest = new OpenPointInTimeRequest(
+            indices, OpenPointInTimeRequest.DEFAULT_INDICES_OPTIONS, TimeValue.timeValueMinutes(randomLongBetween(1, 10)),
+            randomAlphaOfLength(5), randomAlphaOfLength(5)
+        );
+        final Authentication authentication = createAuthentication(new User("test user"));
+        mockEmptyMetadata();
+        final String requestId = AuditUtil.getOrGenerateRequestId(threadContext);
+        if (hasLocalIndices) {
+            assertThrowsAuthorizationException(
+                () -> authorize(authentication, OpenPointInTimeAction.NAME, openPointInTimeRequest),
+                "indices:data/read/open_point_in_time", "test user"
+            );
+            verify(auditTrail).accessDenied(eq(requestId), eq(authentication),
+                                            eq("indices:data/read/open_point_in_time"), eq(openPointInTimeRequest),
+                                            authzInfoRoles(Role.EMPTY.names()));
+        } else {
+            authorize(authentication, OpenPointInTimeAction.NAME, openPointInTimeRequest);
+            verify(auditTrail).accessGranted(eq(requestId), eq(authentication), eq(OpenPointInTimeAction.NAME), eq(openPointInTimeRequest),
+                                             authzInfoRoles(Role.EMPTY.names()));
+        }
+        verifyNoMoreInteractions(auditTrail);
+    }
+
+    public void testUserWithNoRolesCanClosePointInTime() {
+        final ClosePointInTimeRequest closePointInTimeRequest = new ClosePointInTimeRequest(randomAlphaOfLength(8));
+        final Authentication authentication = createAuthentication(new User("test user"));
+        mockEmptyMetadata();
+        final String requestId = AuditUtil.getOrGenerateRequestId(threadContext);
+        authorize(authentication, ClosePointInTimeAction.NAME, closePointInTimeRequest);
+        verify(auditTrail).accessGranted(eq(requestId), eq(authentication), eq(ClosePointInTimeAction.NAME), eq(closePointInTimeRequest),
```

Comment:
```suggestion
        verify(auditTrail).accessGranted(eq(requestId), eq(authentication), eq(indices:data/read/close_point_in_time), eq(closePointInTimeRequest),
```
If you wanna be consistent with the previous test.